packages/perseus/src/widgets/interactive-graphs/graphs/use-transform.ts

```diff
@@ -1,10 +1,39 @@
-import {useTransformContext, vec} from "mafs";
-
-export const useTransform = (...points: vec.Vector2[]) => {
-    const {viewTransform, userTransform} = useTransformContext();
-    const transformToPx = vec.matrixMult(viewTransform, userTransform);
-    const transformedPoints = points.map((point) =>
-        vec.transform(point, transformToPx),
-    );
-    return transformedPoints;
+import {type Interval, vec} from "mafs";
+
+import useGraphConfig from "../reducer/use-graph-config";
+
+const matrixBuilder = vec.matrixBuilder;
+
+export type GraphDimensions = {
+    range: [Interval, Interval];
+    width: number; // pixels
+    height: number; // pixels
+};
+
+export function vectorToPixel(
+    points: vec.Vector2[],
+    graphState: GraphDimensions,
+    translation: vec.Vector2 = [0, 0],
+) {
+    const {range, width, height} = graphState;
+    const [[xMin, xMax], [yMin, yMax]] = range;
+    const transformToPx = matrixBuilder()
+        .translate(...translation)
+        .scale(width / (xMax - xMin), -height / (yMax - yMin))
+        .get();
+    return points.map((p) => vec.transform(p, transformToPx));
+}
+
+export function pointToPixel(
+    points: vec.Vector2[],
+    graphState: GraphDimensions,
+) {
+    const [[xMin], [, yMax]] = graphState.range;
+    const [a, b] = [-xMin, -yMax];
+    return vectorToPixel(points, graphState, [a, b]);
+}
```

Comment:
Added some comments to both vectorsToPixels and pointToPixel. I used the wording you provided; thank you so much! I don't think I actually understood this difference, so I appreciate you clarifying. Please let me know if you think useTransformVectorsToPixels also needs a comment. I'm assuming the other two needed comments because they were harder to differentiate though.